js/src/wp-seo-term-scraper.js

```diff
@@ -1,367 +1,363 @@
-		termSlugInput.on( 'change', updatedTermSlug );
-	}
-
-	/**
-	 * Function to handle when the user updates the term slug
-	 */
-	function updatedTermSlug() {
-		snippetPreview.setUrlPath( termSlugInput.val() );
-	}
-
-	jQuery( document ).ready(function() {
-		var args, termScraper, translations;
-
-		insertTinyMCE();
-
-		$( '#wpseo_analysis' ).after( '<div id="yoast-seo-content-analysis"></div>' );
-
-		tabManager = new TabManager({
-			strings: wpseoTermScraperL10n,
-			focusKeywordField: '#wpseo_focuskw'
-		});
-		tabManager.init();
-
-		termScraper = new TermScraper();
-
-		args = {
-			// ID's of elements that need to trigger updating the analyzer.
-			elementTarget: [ 'content', 'yoast_wpseo_focuskw', 'yoast_wpseo_metadesc', 'excerpt', 'editable-post-name', 'editable-post-name-full' ],
-			targets: {
-				output: 'wpseo_analysis',
-				contentOutput: 'yoast-seo-content-analysis',
-				snippet: 'wpseo_snippet'
-			},
-			callbacks: {
-				getData: termScraper.getData.bind( termScraper ),
-				bindElementEvents: termScraper.bindElementEvents.bind( termScraper ),
-				saveScores: termScraper.saveScores.bind( termScraper ),
-				saveContentScore: termScraper.saveContentScore.bind( termScraper ),
-				saveSnippetData: termScraper.saveSnippetData.bind( termScraper )
-			},
-			locale: wpseoTermScraperL10n.locale
+			tabManager.updateKeywordTab ( score, keyword );
+
+			$ ( '.yst-traffic-light' )
+				.attr ( 'class', 'yst-traffic-light ' + indicator.className )
+				.attr ( 'alt', indicator.screenReaderText );
+
+			$ ( '.adminbar-seo-score' )
+				.attr ( 'class', 'wpseo-score-icon adminbar-seo-score ' + indicator.className )
+				.attr ( 'alt', indicator.screenReaderText );
 		};
 
-		translations = wpseoTermScraperL10n.translations;
+		/**
+		 * Saves the content score to a hidden field
+		 *
+		 * @param {number} score The score calculated by the content assessor.
+		 */
+		TermScraper.prototype.saveContentScore = function( score ) {
+			tabManager.updateContentTab ( score );
+
+			$ ( '#hidden_wpseo_content_score' ).val ( score );
+		};
 
-		if ( translations.length > 0 ) {
-			translations.domain = 'js-text-analysis';
-			translations.locale_data['js-text-analysis'] = translations.locale_data['wordpress-seo'];
-			delete( translations.locale_data['wordpress-seo'] );
+		/**
+		 * Initializes keyword tab with the correct template
+		 */
+		TermScraper.prototype.initKeywordTabTemplate = function() {
+			// Remove default functionality to prevent scrolling to top.
+			$ ( '.wpseo-metabox-tabs' ).on ( 'click', '.wpseo_tablink', function( ev ) {
+				ev.preventDefault ();
+			} );
+		};
 
-			args.translations = translations;
+		/**
+		 * add new descriptionfield to content, creates new element via wp_editor and appends this to the term-description-wrap
+		 * this way we can use the wp tinyMCE editor on the descriptionfield.
+		 */
+		var insertTinyMCE = function() {
+			//gets the textNode from the original textField.
```

Comment:
Please start the comment with a capital and a space between // and the sentence.